packages/rest/README.md

```diff
@@ -12,84 +12,161 @@ npm install @cadl-lang/rest
+| SingletonResourceRead&lt;TSingleton, TResource, TError>                 | Singleton resource GET operation.                                                                                    |
+| SingletonResourceUpdate&lt;TSingleton, TResource, TError>               | Singleton resource PATCH operation.                                                                                  |
+| SingletonResourceOperations&lt;TSingleton, TResource, TError>           | Combines resource GET + PATCH operations                                                                             |
+| ExtensionResourceRead&lt;TExtension, TResource, TError>                 | Extension resource GET operation.                                                                                    |
+| ExtensionResourceCreateOrUpdate&lt;TExtension, TResource, TError>       | Extension resource PUT operation                                                                                     |
+| ExtensionResourceCreate&lt;TExtension, TResource, TError>               | Extension resource POST operation                                                                                    |
+| ExtensionResourceUpdate&lt;TExtension, TResource, TError>               | Extension resource PATCH operation                                                                                   |
+| ExtensionResourceDelete&lt;TExtension, TResource, TError>               | Extension resource GET operation                                                                                     |
+| ExtensionResourceList&lt;TExtension, TResource, TError>                 | Extension resource LIST operation which is a GET operation from a parent resource.                                   |
+| ExtensionResourceInstanceOperations&lt;TExtension, TResource, TError>   | Combines extension resource GET + PATCH + DEL operations.                                                            |
+| ExtensionResourceCollectionOperations&lt;TExtension, TResource, TError> | Combines extension resource POST + LIST operations.                                                                  |
+| ExtensionResourceOperations&lt;TExtension, TResource, TError>           | Combines extension resource instance and collection operations. Includes GET + PATCH + DEL + POST + LIST operations. |
 
 ## See also
 
+- [HTTP example](https://cadlplayground.z22.web.core.windows.net/?c=aW1wb3J0ICJAY2FkbC1sYW5nL3Jlc3QiOwoKQHNlcnZpY2VUaXRsZSgiV2lkZ2V0IFPGFSIpCm5hbWVzcGFjZSBEZW1vxxg7CnVzaW5nIENhZGwuSHR0cDsKCm1vZGVsIMdAewogIEBrZXkgaWQ6IHN0cmluZzsKICB3ZWlnaHQ6IGludDMyxBFjb2xvcjogInJlZCIgfCAiYmx1ZSI7Cn0KCkBlcnJvcsdWRcQMxVVjb2Rly0BtZXNzYWdlymR9CgppbnRlcmbkALLmAI3nALTFP0DkAJ1saXN0KCk6xx9bXSB8xmHEUUByb3V0ZSgid8Uccy97aWR9IinGOHJlYWQoQHBhdGjrANfJSM1GcG9zdCBjcmVhdGUoQGJvZHkgxAXIK9Y0x3pjdXN0b21HZXTId8kR6gC0yjh9Cg%3D%3D):
```

Comment:
Wondering here if we should have a way to open a specific sample in the playground instead of passing the entire code. As this link is doomed to become outdated and be invalid Cadl.
(Out of scope of this PR but maybe we should create an issue for it)

Reply:
They are to the specific samples, hence the long URL. :-)